SuffolkLITLab/courtformsonline.org
# Add deployment workflow and set up next for build static output

This PR is to set up next to output a static build and prepare github pages for deployment.  

Changes:
- Add deploy.yaml github workflow
- Configure workflow to deploy every 24 hours or manually triggered via action tab
- Next config to allow for static build output
- Minor tweaks to current dynamic routes in [topic]
- Changed typescript version for compatablity with eslint 5.4.5 -> 5.4.0

QA steps
- Run `yarn build` locally and verify that it builds successfully
- Verify that static files are output to the /out directory

<img width="630" alt="image" src="https://github.com/SuffolkLITLab/courtformsonline.org/assets/61624970/c1532364-6023-4437-8d5c-53a6cd70db7f">


Note, pages and actions still need configured, this just lays the groundwork for those workflow files
